lib/Sanger/CGP/CavemanPostProcessing.pm

```diff
@@ -410,6 +430,26 @@ sub _get_soft_clip_count_from_cigar{
 	return $count;
 }
 
+sub _getDistanceFromGapInRead{
+  my ($cigar_array,$rdPosIndexOfInterest) = @_;
+  my $min_gap_dist = -1;
+  my $currentRp = 0;
+  foreach my $cigSect(@{$cigar_array}){
+    if($cigSect->[0] eq $MATCH_CIG || $cigSect->[0] eq $SKIP_CIG ||
+          $cigSect->[0] eq $INS_CIG || $cigSect->[0] eq $SOFT_CLIP_CIG){
+      $currentRp+=$cigSect->[1];
+    }elsif($cigSect->[0] eq $DEL_CIG){
```

Comment:
I'm a little confused here.  In the specification (rt721770) the language is a combination of:

* near in/deletions
* alignment gap
* deletion/gap

I'm not sure if you've clarified but I think this is for both insertions and deletions.

Reply:
Question asked in rt721770

Reply:
Insertions and deletions are included